packages/api/src/services/actions.ts

```diff
@@ -79,6 +79,17 @@ export function Handlers(config: Config, appState: Dependencies): Actions {
       // convert this to tuple to save bytes.
       return results.map(({ price, timestamp }) => [timestamp, price]);
     },
+    async getEmpStats(address: string, currency: CurrencySymbol = "usd") {
+      assert(address, "requires address");
+      assert(currency, "requires currency");
+      assert(stats[currency], "No stats for currency: " + currency);
+      return stats[currency].latest.get(address);
+    },
+    async listEmpStats(currency: CurrencySymbol = "usd") {
+      assert(currency, "requires currency");
+      assert(stats[currency], "No stats for currency: " + currency);
+      return stats[currency].latest.values();
+    },
   };
 
   // list all available actions
```

Comment:
would it be useful to have a method where you can feed in an array of EMP addresses and it returns all TVL/stats for that given array?

Reply:
you read my mind, this is actually in a [future PR](https://github.com/UMAprotocol/protocol/pull/3090)